### PR TITLE
Feature 55 about screen

### LINKED ISSUE #55: About screen
As a user i would like to have about screen that describes the purpose of the application so i could read about the application from that

### PR DESCRIPTION
**Pull request by Valmetti**
Resolves #55

### CODE REVIEW

File: src/components/generic/header/header.tsx [hunk 1]
```diff
@@ -41,23 +41,26 @@ class MobileTopBar extends React.Component<Props, State> {
       <>
         <AppBar position="fixed" className={ classes.appBar }>
           <Toolbar>
-            <LogoIcon color="primary" />
-            <Box ml={ 2 } mr={ 2 }>
-              <Typography variant="h1" color="primary">Green paths</Typography>
+            <Box display='flex' flexGrow={1}>
```
Comment (Kaemppi) [inline src/components/generic/header/header.tsx:44]:
```suggestion
            <Box display="flex" flexGrow={ 1 }>
```

File: src/components/screens/about-screen/about-screen.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+  /**
+   * Component render method
+   */
+  public render() {
+    const { classes } = this.props;
+    return (
+      <AppLayout>
+        <Container>
+        <Typography className={ classes.title } variant="h3">
```
Comment (Kaemppi) [inline src/components/screens/about-screen/about-screen.tsx:50]: mind the indentation

**Review by Kaemppi — COMMENTED** [review]
Looking good to me. Just fix the indentation on about-screen.tsx.

File: src/components/generic/header/header.tsx [hunk 3]
```diff
@@ -146,6 +149,24 @@ class MobileTopBar extends React.Component<Props, State> {
...
+            color="primary"
+            variant="text"
+          >
+            { strings.about }
+          </Button>
+        </Link>
+      </Box>
+    );
+  }
+
```
Comment (Daniilmass) [inline src/components/generic/header/header.tsx:169]:
```suggestion
  }
```

Comment (Daniilmass) [inline src/components/generic/header/header.tsx:169]:
```suggestion
    );
  }
```

File: src/components/screens/about-screen/about-screen.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+import { connect } from "react-redux";
+import { NullableToken } from "../../../types";
+import strings from "../../../localization/strings";
+import { Container, Typography, WithStyles, withStyles } from "@material-ui/core";
+import AppLayout from "../../layouts/app-layout/app-layout";
+import { ReduxActions, ReduxState } from "../../../store";
+import styles from "../../../styles/screens/about-screen"
+
+
+/**
```
Comment (Daniilmass) [inline src/components/screens/about-screen/about-screen.tsx:13]:
```suggestion
/**
```
No need for two spaces here

File: src/components/screens/about-screen/about-screen.tsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+        </Typography>
+        <Typography className={ classes.descriptionText } variant="body2">
+          { strings.aboutScreen.descriptionText }
+        </Typography>
+        </Container>
+      </AppLayout>
+    );
+  }
+
+}
```
Comment (Daniilmass) [inline src/components/screens/about-screen/about-screen.tsx:64]:
```suggestion
  }
}
```

**Review by Daniilmass — CHANGES_REQUESTED** [review]
Minor comments

**Pull request merged**